docs/clusterdefinition.md

```diff
@@ -30,51 +30,53 @@ Here are the valid values for the orchestrator types:
+| serviceCidr                     | no       | IP range for Service IPs, Default is "10.0.0.0/16". This range is never routed outside of a node so does not need to lie within clusterSubnet or the VNET                                                                                                                                                                                                                                                                                                            |
+| useInstanceMetadata             | no       | Use the Azure cloudprovider instance metadata service for appropriate resource discovery operations. Default is `true`                                                                                                                                                                                                                                                                                                                                               |
+| useManagedIdentity              | no       | Includes and uses MSI identities for all interactions with the Azure Resource Manager (ARM) API. Instead of using a static service principal written to /etc/kubernetes/azure.json, Kubernetes will use a dynamic, time-limited token fetched from the MSI extension running on master and agent nodes. This support is currently alpha and requires Kubernetes v1.9.1 or newer. (boolean - default == false)                                                        |
+| loadBalancerSku                 | no       | Sku of Load Balancer and Public IP. Candidate values are: `basic` and `standard`. If not set, defaults to `basic`. `standard` requires Kubernetes 1.11 or newer.                                                                                                                                                                                                                                                                                                     |
+| excludeMasterFromStandardLB     | no       | Excludes master nodes from standard load balancer. Requires `loadBalancerSku` to be set to `standard` and Kubernetes v1.11 or newer. Default is `true`.                                                                                                                                                                                                                                                                                                              |

```

Comment:
I still think that this default behavior is counter intuitive since we are always defaulting to true even though that is only supported for k8s 1.11 with standard LB.... which means that we are defaulting to a non-supported option

Reply:
should we flip around the setting to `includeMasterInStandardLB`? @feiskyer thoughts?

Reply:
Could we set `excludeMasterFromStandardLB` only when it is configured explicitly by the user? The default `true` is recommended.

`includeMasterInStandardLB` is actually same with `excludeMasterFromStandardLB`, just with a different default value. Since it's a k8s configure option, it's better to keep it as is.